configure.ac

```diff
@@ -1093,7 +1093,8 @@ AS_IF([test "x$with_gnutls" != "xno"],[
     ## by testing for a 3.4.0+ function which we use
     AC_CHECK_LIB(gnutls,gnutls_pcert_export_x509,[LIBGNUTLS_LIBS="-lgnutls"])
   ])
-  AC_CHECK_HEADERS(gnutls/gnutls.h gnutls/x509.h gnutls/abstract.h)
+  # if any of the required headers is not found, signal we can't support gnutls
+  AC_CHECK_HEADERS(gnutls/gnutls.h gnutls/x509.h gnutls/abstract.h, [], [LIBGNUTLS_LIBS=""])
```

Comment:
These lines look misplaced to me: They should go inside the AS_IF statement above because we should not be even checking GnuTLS headers presence if the admin told us not to enable GnuTLS.

Reply:
Er, they are. The close-brace `])` above them is from `PKG_CHECK_MODULES`.

Reply:
My bad, resolving.